CHANGES.md

```diff
@@ -13,6 +13,7 @@
 ### Preview style
 
 - Fix merging implicit multiline strings that have inline comments (#3956)
+- Improve multiline dictionary and list indentation for sole function parameter (#3964)
```

Comment:
```suggestion
- Multiline dictionaries and lists that are the sole argument to a function are now indented less (#3964)
```
The changelog entry should just describe the change; it's up to the reader to judge whether it's an improvement :)